sljit_src/sljitNativeRISCV_common.c

```diff
@@ -140,7 +164,13 @@ static const sljit_u8 freg_map[SLJIT_NUMBER_OF_FLOAT_REGISTERS + 3] = {
 #define SW		(F3(0x2) | OPC(0x23))
 #define XOR		(F7(0x0) | F3(0x4) | OPC(0x33))
 #define XORI		(F3(0x4) | OPC(0x13))
-
+#if defined __riscv_zbb
+#if defined SLJIT_CONFIG_RISCV_32
+#define ZEXTH		(IMM_I(0x80) | F3(0x4) | OPC(0x33))
+#elif defined SLJIT_CONFIG_RISCV_64
+#define ZEXTH		(IMM_I(0x80) | F3(0x4) | OPC(0x3B))
```

Comment:
Does the instruction form change? The register size should not matter.

Reply:
Yes.

https://drive.google.com/file/d/1uviu1nH-tScFfgrovvFCrj7Omv8tFtkp/view page 266.